v2.0/partitioning.md

```diff
@@ -0,0 +1,371 @@
+
+~~~ shell
+# Set enterprise license
+$ SET CLUSTER SETTING cluster.organization = 'organization_name';
+$ SET CLUSTER SETTING enterprise.license = 'xxxxxxxxxxxx'; 
+
+# Partition a table by list (geo-partitioning)
+$ CREATE TABLE <table-name> ( <elements...>, PRIMARY KEY (<primary_keys>))
+  PARTITION BY LIST(primary_key_component) (PARTITION <partition-name> VALUES IN ( <list-expr>... ));
+
+# Partition a table by range (archival-partitioning)
+$ CREATE TABLE <table-name> ( <elements...>, PRIMARY KEY (<primary_keys>)) 
+  PARTITION BY RANGE(primary_key_component) (PARTITION <partition-name> VALUES FROM <range-expr> TO <range-expr>);
+
+# Create/edit the replication zone for a partition:
+$ cockroach zone set <database.table.partition> --file=<zone-content.yaml> <flags>
+
+# Remove the replication zone for a partition:
+$ cockroach zone rm <database.table.partition> <flags>
+~~~
+
+### Verify Table Partitions
+
+To verify the table partitions, use the [`SHOW CREATE TABLE`](show-create-table.html) statement.
+
+## Scenario-based Examples
+
+### Define Table Partitions by List
+
+Consider a global online learning portal, RoachLearn, that has a database containing a table of students across the world. Suppose we have two datacenters: one in the United States and another in Australia. To reduce latency, we want to keep the students' data closer to their locations: 
+
+- We want to keep the data of the students located in the United States and Canada in the United States datacenter.
+- We want to keep the data of students located in Australia and New Zealand in the Australian datacenter. 
+
+We can achieve this by partitioning on `country` and using the `PARTITION BY LIST` syntax. 
+
+#### Step 1: Start each node with its datacenter location specified in the `--locality` flag:
+
+~~~ shell
+# Start the node in the US datacenter:
+$ cockroach start --insecure --locality=region=us-1  --store=node1 --host=<node1 hostname> --port=26257 --http-port=8080 --join=<node1 hostname>:26257,<node2 hostname>:26258
+
+# Start the node in the AUS datacenter:
+$ cockroach start --insecure --locality=region=aus-1 --store=node2 --host=<node2 hostname> --port=26258 --http-port=8081 --join=<node1 hostname>:26257,<node2 hostname>:26258
+~~~
+
+#### Step 2: Set the enterprise license
+
+To set the enterprise license, see [Set the Trial or Enterprise License Key](enterprise-licensing.html#set-the-trial-or-enterprise-license-key)
+
+#### Step 3: Create a table with the appropriate partitions:
+
+~~~ sql
+> CREATE TABLE students_by_list (
+    id SERIAL,
+    name STRING,
+    email STRING,
+    country STRING,
+    expected_graduation_date DATE,   
+    PRIMARY KEY (country, id))
+    PARTITION BY LIST (country)
+      (PARTITION north_america VALUES IN ('CA','US'), 
+      PARTITION australia VALUES IN ('AU','NZ'), 
+      PARTITION DEFAULT VALUES IN (default));
+~~~
+
+#### Step 4: Create corresponding zone configurations:
+
+~~~ shell
+$ cat > north_america.zone.yml
```

Comment:
Are these commands supposed to just show the contents of the files? If so, let's split the command from the response. 

Also, I don't think you need the `>`. You can just do `cat north_america.zone.yml`. 

Same applies below.